src/generateSdk/astGeneratorHandler.ts

```diff
@@ -27,6 +27,7 @@ interface AstGeneratorPlugin {
 export async function generateAst(
     input: AstGeneratorInput,
     plugins: string[] | undefined,
+    backendPath?: string,
```

Comment:
I would name it `rootPath` everywhere.
```suggestion
    rootPath?: string,
```

Reply:
Also, what if we include this path inside `AstGeneratorInput`? this way we could have this nice `AstGeneratorInput -> AstGeneratorOutput`, instead of `(AstGeneratorInput, path) -> AstGeneratorOutput`

Reply:
i would find it a bit confusing to name it root since we also have the notion of the root of the mono repo project. i personally think of that as root, event though in the context of AST generation the "root of the action" is the backend folder.
i will try to see if I can add the value as a field in the `AstGeneratorInput` class